python/orca/src/bigdl/orca/learn/pytorch/experimential/mmcv/mmcv_ray_estimator.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import types
+import copy
+from bigdl.dllib.utils.log4Error import *
+
+from typing import (Any, Dict, List, Optional, Tuple, Callable, overload)
+
+from bigdl.orca.learn.pytorch.experimential.core.base_ray_estimator import BaseRayEstimator
+from bigdl.orca.learn.pytorch.experimential.mmcv.mmcv_ray_runner import MMCVRayRunner
+
+
+class MMCVRayEstimator(BaseRayEstimator):
+    def __init__(self,
+                 *,
+                 mmcv_runner_creator=None,
+                 runner_cls=MMCVRayRunner,
+                 backend="ray",
+                 workers_per_node=1,
+                 config=None):
+        if not (isinstance(mmcv_runner_creator, types.FunctionType)):
+            invalidInputError(False, "Must provide a function for mmcv_runner_creator")
+
+        self.mmcv_runner_creator = mmcv_runner_creator
+        self.backend = backend
+        self.config = {} if config is None else config
+        worker_config = copy.copy(self.config)
+        params = dict(
+            mmcv_runner_creator=self.mmcv_runner_creator,
+            config=worker_config
+        )
+        self.setup(params, self.backend, runner_cls, workers_per_node)
+        self.num_workers = len(self.remote_workers)
+
+    def fit(self,
```

Comment:
Wrap fit() with another method called `run(self, data_loaders_creators)`, for being consistent with mmcv runner, or try to just add `self.run=self.fit` in __init__()?